examples/retrievals/plot_skewt.py

```diff
@@ -8,9 +8,9 @@
 """
 
 
```

Comment:
Can we have this in plotting as well?  We do some retrievals here but not sure it's the focus of it.